apps/mobile/src/components/widgets/collectibles/collectibles-serializer.ts

```diff
@@ -0,0 +1,21 @@
+import { type Collectible, type CollectibleCardProps } from '@leather.io/ui/native';
```

Comment:
I added this as a way to just get only the data we need to display from the mocks. 

Later I found `createBestInSlotInscription` which is actually giving us this data and more. 

I will investigate more when hooking up APIs but maybe it could be good to have a BE proxy for this type of stuff - data we need for App Home for example.